superset/config.py

```diff
@@ -194,6 +194,23 @@
 # will result in combined feature flags of { 'FOO': True, 'BAR': True, 'BAZ': True }
 DEFAULT_FEATURE_FLAGS = {}
```

Comment:
I think we need some guidance here on what should be in a feature flag and what should be like a variable in this config file. 

Per my understanding: If you have anything JS facing, it should be a feature flag. But what about backend only changes ? We don't *have* to use a feature flag for them ? 

Some guidance to clarify this would be great.

Reply:
@xtinec was suggesting to have both `FEATURE_FLAGS` and `FEATURE_CONF`, to me I think it'd be simpler to have a single dict that acts as both. I vote for having just one. If that's the case, we may want to either accept that the term "flag" be overloaded with configurations, or quickly change the term flag to conf.